### ISSUE TITLE
Should tanka deploy test charts by default?

### ISSUE THREAD

**Issue by gassiss** [grafana/tanka#550]
Hi team!

It seems like when I run `tk apply` it automatically deploys test charts as well. Is that intended behaviour? I feel like it should only run tests if we explicitly tell it to.

I put together a small repo reproducing this: https://github.com/gassiss/tanka-issue-example

**Comment by Duologic** [comment]
How does Helm handle this? Tanka simply renders the chart with `helm template`, this output can be modified in jsonnet.

**Comment by lkiii** [comment]
```jsonnet
local excludeChartTests(resources) =
  {
    [item.key]: item.value
    for item in std.objectKeysValues(resources)
    if !(std.type(item.value) == 'object' &&
       std.objectHas(item.value, 'metadata') &&
       std.objectHas(item.value.metadata, 'annotations') &&
       std.objectHas(item.value.metadata.annotations, 'helm.sh/hook') &&
       item.value.metadata.annotations['helm.sh/hook'] == 'test')
  };
```

I was having the same issue and this looks like something universal which could help in this case

**Comment by zerok** [comment]
`helm template` also has a `--skip-tests` flag. Perhaps it's just enough that we use that flag when running `helm template` ?

**Comment by zerok** [comment]
Using the `skipTests` property when calling `helm.template` in your wrapper code should work (at least it passes the right arguments):

```
helm.template('nginx', './charts/nginx-ingress', {
  skipTests: true,
  // values: { ... }
}),
```

**Comment by lkiii** [comment]
Yep it works well with the flag on my end as well. https://stackoverflow.com/a/79346465/816230 I have posted the example here

**Comment by strowi** [comment]
Can confirm,  @zerok 's  `skiptests: true` works (not case-sensitive). 
Had to search through github issues to the MR, this probably could be better documented on tanka.dev ;)

**Comment by zerok** [comment]
@strowi https://github.com/grafana/tanka/pull/1329 first try 🤣
